core/modules/image/image.field.inc

```diff
@@ -737,6 +773,19 @@ function image_field_formatter_settings_summary($field, $instance, $view_mode) {
     $summary[] = t('Loading: @speed', array('@speed' => $settings['image_load']));
   }
 
+  $formatter_settings = $instance['display'][$view_mode]['settings'];
+
+  if (!empty($formatter_settings['image_svg_settings']['width'])
+    || !empty($formatter_settings['image_svg_settings']['width'])) {
+    $summary[] = t(
+      'SVG images attributes: <br /> Width: @widthpx,<br /> Height: @heightpx',
+      array(
+        '@width' => (int) $formatter_settings['image_svg_settings']['width'],
+        '@height' => (int) $formatter_settings['image_svg_settings']['height'],
```

Comment:
...as per my previous comment, these do not match the placeholders used in `t()`.

Also, if we use number fields instead of textfields, we might not need to be explicitly type casting here.

Reply:
Fixed